app/services/notifications/create_from_model_service.rb

```diff
@@ -292,7 +292,13 @@ def add_receiver(receivers, collection, reason)
   end
 
   def remove_self_recipient(receivers)
-    receivers.delete(user_with_fallback.id)
+    if receivers.key?(user_with_fallback.id)
+      self_reasons = receivers[user_with_fallback.id]
+      self_reasons.delete_if { |item| item != :mentioned }
```

Comment:
Please use the constant here: `NotificationSettings::MENTIONED`

Reply:
done